lib/omniauth-artsy/version.rb

```diff
@@ -2,6 +2,6 @@
 
 module Omniauth
   module Artsy
-    VERSION = '0.2.3'
+    VERSION = '0.3.3'
```

Comment:
Maybe bumping this up to `'0.3.0'` makes more sense for a minor version upgrade?